study-builder/studies/prion/medical.conf

```diff
@@ -512,19 +490,19 @@
                     "exclusive": false
                   },
                   {
-                    "stableId": "AO",
+                    "stableId": "AX",
                     "optionLabelTemplate": {
                       "templateType": "TEXT",
                       "templateCode": null,
-                      "templateText": "$prion_medical_country_ao",
+                      "templateText": "$prion_medical_country_ax",
                       "variables": [
                         {
-                          "name": "prion_medical_country_ao",
+                          "name": "prion_medical_country_ax",
                           "translations": [
-                            { "language": "en", "text": ${i18n.en.prion_medical_country_ao} },
-                            { "language": "es", "text": ${i18n.es.prion_medical_country_ao} },
-                            { "language": "he", "text": ${i18n.he.prion_medical_country_ao} },
-                            { "language": "zh", "text": ${i18n.zh.prion_medical_country_ao} }
+                            { "language": "en", "text": ${i18n.en.prion_medical_country_ax} },
+                            { "language": "es", "text": ${i18n.es.prion_medical_country_ax} },
+                            { "language": "he", "text": ${i18n.he.prion_medical_country_ax} },
+                            { "language": "zh", "text": ${i18n.zh.prion_medical_country_ax} }
```

Comment:
If we're only ever going to use the English names for the countries, why not have those names in one file and use it for all the languages?

```
{ "language": "en", "text": ${i18n.country.ax} },
{ "language": "es", "text": ${i18n.country.ax} },
{ "language": "he", "text": ${i18n.country.ax} },
...
```

Reply:
We are going to use the names in each language--I just got lazy and didn't stick a prefix in front of all the countries because we just submitted the language files for translation so soon enough I'll be able to retest with the actual language files.